index.js

```diff
@@ -14,14 +17,52 @@ function sortByStringLength() {}
  * Example: "Hello"
  * [ 'elloH', 'lloHe', 'loHel', 'oHell', 'Hello' ]
  */
-function textScroller() {}
+
+function textScroller(word) {
+  const scrolledWords = [];
+
+  for (const char of word) {
```

Comment:
Nice work here ya'll but I would actually recommend that if we make a function that accepts a parameter like,`word`, we create a copy of the variable passed to the function rather than directly manipulate the data that is passed to our function